### PR TITLE
squashing: recommend --keep-base when squashing without a conflict

### PR DESCRIPTION
**Pull request by RalfJung**

### CODE REVIEW

**Review by jieyouxu — APPROVED** [review]
Cool

**Comment by jieyouxu** [comment]
(You can self-merge, we don't have the 1-approval requirement anymore if u have write access)

**Comment by RalfJung** [comment]
I could but that doesn't mean I should. I'd rather have some double-check my edits and make sure there's not some context I am overlooking.

**Pull request merged**
